src/pages/library/Datasets.js

```diff
@@ -107,6 +109,16 @@ const Participant = ({ logo, title, shortDescription, description, sizeText, mod
 
 const browseTooltip = 'Look for the Export to Terra icon to export data from this provider.'
 
+const captureEventMetrics = (eventName, metrics) => {
+  Ajax().Metrics.captureEvent(eventName, metrics)
+}
+
+const captureBrowseDataEvent = (datasetName) => {
+  const eventName = `${Events.datasetLibraryBrowseData}`
+  captureEventMetrics(eventName, {
+    datasetName: datasetName
+  })
+}
```

Comment:
Equivalent, and doesn't swallow the promise returned by captureEvent (which may be useful to the caller):
```suggestion
const captureBrowseDataEvent = datasetName =>
  Ajax().Metrics.captureEvent(Events.datasetLibraryBrowseData, {datasetName})
```